### PR TITLE
Build the Ubuntu core full disk encryption takeover

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build the Ubuntu core full disk encryption takeover with animation.

**Missing landing page link**

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to the homepage
- Reload until you see the Full disk encryption takeover
- Check the copy matches [the copy doc](https://docs.google.com/document/d/1r9wlmgwrkFahKHWi-Vf6xE8YqIRSklGWY4o-0w6ytQ4/edit?ts=5c580bd9#heading=h.meo0azuz09li)
- Check the design matches [the visual design](https://github.com/ubuntudesign/web-squad/issues/968)
- Check the animation makes you smile :smile: 

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/969

### CODE REVIEW

**Comment by anthonydillon** [comment]
@anasereijo Added a delay to the animation.

**Comment by anasereijo** [comment]
- [ ] Check the animation makes you smile 💯 💯 💯

File: templates/takeovers/_ubuntu-core-full-disk-encryption.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<section lang="en" class="js-takeover is-bordered p-takeover p-strip--image is-deep">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h1>Secure your IoT device’s data</h1>
+      <p class="p-heading--four u-sv2">
+        Learn how Ubuntu Core with full disk encryption and secure boot protects device data from physical access.
+      </p>
+      <p class="u-hide--small">
+        <a itemprop="url" href="#" class="p-button--positive">Download the whitepaper now</a>
```
Comment (pmahnke) [inline templates/takeovers/_ubuntu-core-full-disk-encryption.html]: missing link

Comment (caldav) [inline templates/takeovers/_ubuntu-core-full-disk-encryption.html]:
```suggestion
        <a itemprop="url" href="/engage/iot-disk-encryption?utm_source=takeover&utm_campaign=CY19_IOT_Ubuntu Core_Whitepaper_Secure booth/FDE" class="p-button--positive">Download the whitepaper now</a>
```

File: templates/takeovers/_ubuntu-core-full-disk-encryption.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                          <path d="M66.4182453,41 C65.2149456,41 64.2340724,40.0096702 64.2219899,38.7780703 L64.0001127,16.2664434 C63.9935222,15.6015478 64.2763608,14.9675123 64.7722895,14.5354704 C65.2682181,14.1045507 65.9267103,13.919951 66.5670788,14.0321697 L124.173336,24.0936959 C125.368947,24.3024226 126.172978,25.4616415 125.968126,26.6837028 C125.763823,27.9057642 124.625878,28.7266438 123.433013,28.5173559 L68.4200835,18.9091932 L68.6155991,38.7326217 C68.6276815,39.972077 67.6539479,40.9870949 66.4407626,41 C66.4330737,41 66.4253849,41 66.4182453,41" id="Fill-14"></path>
+                      </g>
+                  </g>
+              </g>
+          </g>
+      </svg>
+      <p class="u-hide--large u-hide--medium u-no-margin--bottom" style="margin-top: 1rem; max-width: none;">
+        <a itemprop="url" href="#" class="p-button--positive">Download the whitepaper now</a>
```
Comment (caldav) [inline templates/takeovers/_ubuntu-core-full-disk-encryption.html]:
```suggestion
        <a itemprop="url" href="/engage/iot-disk-encryption?utm_source=takeover&utm_campaign=CY19_IOT_Ubuntu Core_Whitepaper_Secure booth/FDE" class="p-button--positive">Download the whitepaper now</a>
```

**Comment by anthonydillon** [comment]
@pmahnke @caldav good to go?

**Pull request merged**